calculator/src/components/Ouput.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+import { Doughnut } from 'react-chartjs-2';
```

Comment:
I'm assuming this is intended for a more fleshed-out version of this project but is not yet in use? I'm curious to know how you intend to use a doughnut chart to represent the tipping breakdown? Would this include the input of the party member's names? 

I personally left a few lines of CSS code in that I commented out, but I know I should have removed them, to be honest. It is best practice to keep your code as clean as possible by not including any unused code. If you intend to leave it as you'll be returning to it, then I'd suggest adding in a comment stating it's future use. :)